package.json

```diff
@@ -60,7 +60,6 @@
     "@babel/eslint-parser": "^7.22.7",
     "@babel/parser": "^7.22.7",
     "@babel/types": "^7.22.5",
-    "@nrwl/nx-cloud": "16.1.1",
```

Comment:
This was outdated, the package is `nx-cloud` now which we already have specified